### DISCUSSION TITLE
wormhole bridging issue

### DISCUSSION THREAD

**Discussion by scrubmasterara** [wormhole-foundation/wormhole#413] [Problems / bugs] [answered]
Hi! i tried to bridge Link tokens from eth to solana  for dev purposes and it worked flawlessly. However, when i tried to bridge link tokens back from solana to eth, the tokens were successfully burned but i didn't receive any Link into my eth wallet. Is there any way to get the tokens back or am i out of luck?

https://solscan.io/tx/2uxfK8ya9r5ABkoW3u2CunvH1WgXtf8xBPVKP3LeJucNAZzgTHvrmfmUCJDbMEpAmjQBK1kc6em9cTMxEBum1rDr

**Comment by leoluk** [comment, marked as answer]
Wrong transaction, please provide the tx which burned the tokens

**Reply by scrubmasterara** [reply]
is this not the burn transaction. The minting transactions said +100 link. this transaction says -100 link.

**Reply by leoluk** [reply]
You're right - I must've mixed it up.

The transaction was successfully processed by the Wormhole network, but the client-side transaction submission may have failed. Try clearing your cache and reloading wormholebridge.com to see if the transaction can be completed there. Make sure both wallets are connected.

If wormholebridge.com doesn't work, you can manually submit the signed VAA bytes below to the submitVAA method on the Wormhole contract: https://etherscan.io/address/0xf92cd566ea4864356c5491c177a430c222d7e678#writeContract

---

Digest (for reference only): 5721f72d94fdd124e48552cd346f058385bd6c5393200d8ebc27ccc8e0971d94

Signed VAA bytes to submit to the contract:

> 0x01000000010d0090e42b67d1cfef30f1c986bdb49ffb8a7a6ba0c6cc0a29414817843f8b4c3cef0ac7468d2d05df009ac4c5fed7f0c804704763a0b75f5b8772327e7fe960ba6101011d0058be7a563f98b150f16a377bcd7ee3a186bbf0c627e9f31f521a402b01bb708ace49fa13c62c6ba6ec5a5f265d0a57fde740acee4dcdddf2c19774913122010330e8b2512cb3d272318577a718af6e145f181d00c1f8771158ed8ebb506fc12d761edb07536e3ade750eaafe4f4561b4258ae1fe69b27b53c793a3edbee174710105e25d2bcec16b0104b4504a92a2d39d5168e37d875a8233899ea30dafdad4fb564e0aa132bb453324a785b4f6be5de3986be59c849fc5d6f5abb7550c4d7b507800065ee4a8612a05db8219a1d842940945afe136ea9a216ca279cd0cc388270da418186c661904ba7f2b290d31ab6a866b20a82d03d57dfc5f99975da8fa777c469a0007bc65428c710d3da5f08ecab82b239c000ec8c8301b1b44be103dcc1f1128afee0f6ee47dcd3610fbca824f3800ded451aaec26e680609a7395f23e983537e1ee00087aae4f06f69620cb27c822d7e5f3ae2c95cbb4cf768cae1552d5a0abb718e4290565fc4b5d9fdbd07aa4859fe613b6c100af512eb0836cfcad6a50851a0d4cb30009e31fad86b344d21b3c663b523e995b45ec51e47d96aaca236f1452d91980399579689e706fdba2f225a551506bd2deb263c76436e7685f4242e2cca1026b6de2000a90bb94f4a1c7b86d71bcbde825a2a4d3102bdb10ab025481d2a4eaf3978af8cc5f99ce937c2afe81bbfc7a2ac0bd9e78ff3892f9ec948ac2db2abcdb6e1f6e24000b34af9ddc4b07fb39a663d107d5606383ea4e96185e9ef9ffe29ebf11d78ed69174f1afb8040c5b9157f75904f79ec2848aa6498661461a4d0f8c88011ec74844000d497ccef2b853effa0b9be93b9f554c353d5e3c30abea1c52bbcd331b25c8732d6a5a8ee2a80d1f6a3bda3ea0ce7cf211868d81df5ef8dcc8c7ee3f0a3243f7270011e37b14573b57362c46e6822366be5e0912849b4fbdf5c39c8ed47ce166131bfe6abbbdb86bbf00ddd03a44d78037f84ea0dda6a5688f27a5eae98a9eaca15fd10012c431919fb40a05aa661f35f19194e8389e016926edc54feb5b101001a68876ec18bd1360c168a72697519991d280db9671463680eecb9cbc4195709adcc2bedf016139b31210000047b60102f6ff5b2bfaa2b6e50f08576788d2f49073e3756c8fde9056f850e912a2602ea2000000000000000000000000ab0705bd932a3517cb461e39159110834434583e02000000000000000000000000514910771af9ca656af840dff83e8264ecf986ca09000000000000000000000000000000000000000000000000000000174876e800
